### DISCUSSION TITLE
Compiling it as DLL

### DISCUSSION THREAD

**Discussion by locorico** [dlemstra/Magick.NET#609] [General] [answered]
I'm new to VS and .NET development process, apologies for the newbie qeustions.

Basically, I've download the Magick.NET source code (7.16.0.0).
Trying to compile it as a DLL, and then add the DLL to my standalone C# Windows desktop application (Project -> Reference -> add DLL).

Steps I've take:
1) Selected ReelaseQ8 -> x64 -> Magick.NET (framework net20).
2) Build -> Magick.NET

Errors that I got:
CSC : error CS1566: Error reading resource 'ImageMagick.Resources.Xml.colors.xml' -- 'Could not find a part of the path "...\Magick.NET-7.16.0.0\src\Magick.Native\resources\ReleaseQ8\colors.xml'.'

What am I doing wrongly?

**Comment by dlemstra** [comment, marked as answer]
You might want to take a look at `Building.md` in the root of this project. Please let me know if something is missing from those docs. 

And is there a reason you are not using the NuGet package?

**Comment by locorico** [comment]
Skipping the compilling as i'm having issue installing the Magick.Native too.
Went with NuGet and it worked thanks
